ksql-engine/src/test/java/io/confluent/ksql/function/InternalFunctionRegistryTest.java

```diff
@@ -285,6 +286,11 @@ public String getDescription() {
               }
             };
           }
+
+          @Override
+          public List<List<Schema>> supportedArgs() {
+            return ImmutableList.of();
```

Comment:
Is this different from returning `ImmutableList.of(ImmutableList.of())` as in `CountAggFunctionFactory` above to indicate everything is a supported type?

Reply:
Yes - this means "nothing is supported". A little subtle 😄 - I suppose technically the `ImmutableList.of(ImmutableList.of())` means "a single method with no arguments" is supported, but from an implementation perspective it's identical.